proto/beacon/rpc/v1/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+# gazelle:ignore
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+
+go_proto_library(
+    name = "go_grpc_gateway_library",
+    compilers = [
+        "@io_bazel_rules_go//proto:go_grpc",
+        "@grpc_ecosystem_grpc_gateway//protoc-gen-grpc-gateway:go_gen_grpc_gateway",
```

Comment:
Did you want to add this to the gRPC gateway? it's not automatic

Reply:
Added in #5668